src/sdk/pynni/nni/compression/torch/quantization/quantizers.py

```diff
@@ -110,12 +112,111 @@ def get_bits_length(config, quant_type):
     else:
         return config["quant_bits"].get(quant_type)
 
+def _check_weight(module):
+    try:
+        return isinstance(module.weight.data, torch.Tensor)
+    except AttributeError:
+        return False
+
+class Post_Training_Quantize():
+    """Post_Training_Quantize contains two major function. The first is global scale quantization.
+    The second is mixed scale quantization.
+    """
+
+    def __init__(self, model, quant_bits=-1):
+        self.model = model
+        self.quant_bits = quant_bits
+        """
```

Comment:
it is really strange that you put docstring behind the implementation of `__init__`